sdk/javascript/processor/exceptions.js

```diff
@@ -17,16 +17,31 @@
 
 'use strict'
 
-class InvalidTransaction extends Error {
-  constructor (message = '') {
+class _TransactionProcessorError extends Error {
+  constructor (message = '', extendedData = null) {
```

Comment:
Although `extended_data` makes sense in the proto definition, why would the data be "extended" for the person throwing this error? Since it is the only data being included, why not just call it `data`?